operations/commit_queue_test.go

```diff
@@ -230,9 +231,6 @@ func (s *CommitQueueSuite) TestListContentsForPRs() {
 	stringOut := string(out[:])
 
 	s.Contains(stringOut, "Project: mci")
-	s.Contains(stringOut, "Repo: evergreen")
```

Comment:
I removed them because previously they were previously only displayed for PR queue types. Since you have to pass the project for the commit queue as a parameter for this CLI command, I feel like showing info about the project isn't really needed